flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -100,6 +100,22 @@ def read_df(
 
 
 def command_safe_run(command, command_name="mycommand", fail_on_fail=True):
+    """
+    Verifies that a command is valid by attempting to run it. Prints stream of code if command fails.
```

Comment:
Documentation titles should be one sentence/phrase. Maybe something like "Runs a shell command and prints diagnostics if command fails." or similar?